packages/ocaml-r/ocaml-r.0.1.0/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "philippe.veber@gmail.com"
+authors: ["Guillaume Yzyquel" "Maxence Guesdon" "Philippe Veber"]
+homepage: "https://github.com/pveber/ocaml-r"
+bug-reports: "https://github.com/pveber/ocaml-r/issues"
+license: "GPL"
+dev-repo: "https://github.com/pveber/ocaml-r.git"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "base" {build}
+  "configurator" {build}
+  "jbuilder" {build}
+  "ocamlfind"
+  "stdio" {build}
+]
+depexts: [
```

Comment:
Is it possible to use the `conf-r` package instead ? Do you rely on the mathlib version ? (I don't know how it works)

Reply:
Actually I think it'd be safer to put both.